stable/database/README.md

```diff
@@ -238,6 +238,12 @@ The following tables list the configurable parameters of the `database` chart an
 | `persistence.storageClass` | Storage class for volume backing database archive storage | `-` |
 | `configFilesPath` | Directory path where `configFiles.*` are found | `/etc/nuodb/` |
 | `configFiles.*` | See below. | `{}` |
+| `sm.logPersistence.enabled` | Whether to enable persistent storage for logs | `false` |
+| `sm.logPersistence.overwriteBackoff.copies` | How many copies of the crash directory to keep within windowMinutes | `3` |
+| `sm.logPersistence.overwriteBackoff.windowMinutes` | The window within which to keep the number of crash copies | `120` |
+| `sm.logPersistence.accessModes` | Volume access modes enabled (must match capabilities of the storage class) | `ReadWriteOnce` |
+| `sm.logPersistence.size` | Amount of disk space allocated for log storage | `20Gi` |
```

Comment:
If we are expecting core files to be saved, then I think 20Gi will usually be too small.

I would suggest a minimum default would be 60Gi.
Customers can increase that if they have increased the process MEM and/or increased the crashBackoff.copies.

Reply:
Updated